src/sass/app_2/_common/components/accounts_switcher.scss

```diff
@@ -43,23 +43,19 @@
             p.acc-balance-amount {
                 line-height: 100%;
                 color: $COLOR_GREEN_1;
-                font-size: 1rem;
-                padding: 0 16px 0 8px;
+                font-size: 1.6em;
```

Comment:
so we're not going to use variables for fonts?

Reply:
we will, was thinking to add that in another PR